theano/gpuarray/dnn.py

```diff
@@ -164,13 +164,13 @@ def _dnn_check_compile():
 
 def _dnn_check_version():
     v = version()
-    if v < 5000:
-        return False, "cuDNN version is too old. Update to v5* or higher, was %d." % v
-    if v >= 6100:
+    if v < 6000:
+        return False, "cuDNN version is too old. Update to v6* or higher, was %d." % v
+    if v >= 7200:
```

Comment:
why 7200? I would keep doing as before and use 7100. This way user will know we didn't tested that version of Theano with a new release.